cajita/unit_test/tstFastFourierTransform.hpp

```diff
@@ -97,19 +86,53 @@ void forwardReverseTest()
     // Copy to the device.
     Kokkos::deep_copy( lhs_view, lhs_host_view );
 
-    // Create an FFT
-    auto fft = Experimental::createFastFourierTransform<double, TEST_DEVICE>(
-        *vector_layout, Experimental::FastFourierTransformParams{}
-                            .setCollectiveType( 2 )
-                            .setExchangeType( 0 )
-                            .setPackType( 2 )
-                            .setScalingType( 1 ) );
-
-    // Forward transform
-    fft->forward( *lhs );
-
-    // Reverse transform
-    fft->reverse( *lhs );
+    // Create FFT options
+    Experimental::FastFourierTransformParams params;
+
+    // Set MPI communication
+    params.setAllToAll( true );
+
+    // Set data exchange type (false uses slab decomposition)
+    params.setPencils( true );
+
+    // Set data handling (true uses contiguous memory and requires tensor
+    // transposition; false uses strided data with no transposition)
+    params.setReorder( true );
+
+    if ( use_default && use_params )
+    {
+        auto fft =
+            Experimental::createHeffteFastFourierTransform<double, TEST_DEVICE>(
+                *vector_layout, params );
+        // Forward transform
+        fft->forward( *lhs, Experimental::FFTScaleFull() );
+        // Reverse transform
+        fft->reverse( *lhs, Experimental::FFTScaleNone() );
+    }
+    else if ( use_default )
+    {
+        auto fft =
+            Experimental::createHeffteFastFourierTransform<double, TEST_DEVICE>(
+                *vector_layout );
+        fft->forward( *lhs, Experimental::FFTScaleFull() );
+        fft->reverse( *lhs, Experimental::FFTScaleNone() );
+    }
+#ifndef TEST_CUDA
```

Comment:
Why is this here?

Reply:
This section only tests the explicit use of an FFT backend, rather than the default for the Kokkos backend. Since there are only options for the host, I needed to disable this from the automatic test expansion on all Kokkos backends

Reply:
Replaced with a check for any enabled device backend